### ISSUE TITLE
Error: <stdin>: syntax error in line 115 scanning a quoted string (missing endquote? longer than 16384?)

### ISSUE THREAD

**Issue by pramodk** [LLNL/STAT#17]
While using STAT I am seeing below error:

```
Error: <stdin>: syntax error in line 115 scanning a quoted string (missing endquote? longer than 16384?)
String starting:"[0-1,4,7-8,10-12,14-15,17,19-22,24,27,29-32,35,40-42,44-46,48,52-55,57-62,65-70,
```

<img width="1135" alt="image" src="https://user-images.githubusercontent.com/666852/75093843-baba3c00-5585-11ea-8dd6-e8e867164ab5.png">

I assume this is mostly coming from dot (because of too divergent call stacks?) but do you have any suggestions to workaround this?

cc: @matz-e

**Comment by lee218llnl** [comment]
could you perhaps attach the .dot file that triggered this error? It should be in the stat_results directory.

**Comment by pramodk** [comment]
Sure !  Could you check attached zip where we have 00_dplace.0018.2D.dot file? [dplace.0018.zip](https://github.com/LLNL/STAT/files/4247194/dplace.0018.zip)

Let me know if additional information is required.

**Comment by lee218llnl** [comment]
Hmm, I get a different error "Error: bad label format main::{lambda()#1}::operator()() const". My stat-view is the current "develop" branch. Which version are you using? I thought I had fixed issues with long, divergent task lists. If I take the '{}' brackets out of that stack frame, it displays correctly without my error or yours.

**Comment by lee218llnl** [comment]
https://github.com/LLNL/STAT/commit/4317c790a03ea68bbb61d5059e5c318a0b211bf1 should allow stat-view to handle brackets in the node names now. Let me know if you are able to build/install the develop branch and test it yourself.

**Comment by lee218llnl** [comment]
This is also now included in the latest 4.0.2 release https://github.com/LLNL/STAT/releases/tag/v4.0.2. @pramodk can you please test this out?

**Comment by pramodk** [comment]
@lee218llnl : thank you very much!

> Which version are you using?

I was using 3.0.1 because of segfault I mentioned in #15.

>  can you please test this out?

I don't have access to the system for 10 days and hence I will be test this only after this Friday. I will update you as soon as I test this further.

**Comment by lee218llnl** [comment]
OK, the long edge label has definitely been fixed since 3.0.1. Regarding #15, I'm hoping the new version won't seg fault or that you can try letting spack build python and all the necessary site packages. All said, since the error that you reported in this particular issue has been known to have been fixed, I will close this out.
